vhd/packer/install-dependencies.sh

```diff
@@ -351,10 +351,9 @@ K8S_VERSIONS="
 1.19.0-alpha.3
 1.18.2
 1.18.1
+1.17.6
 1.17.5
 1.17.5-azs
-1.17.4
-1.17.4-azs
```

Comment:
@jadarsie Are there any in-code Azure Stack docs we want to update now that we're removing support for 1.17.4?

Reply:
I cannot think of anything besides the azure-stack doc/topic, I can update that page in the next few days.